asdf/tests/helpers.py

```diff
@@ -23,12 +23,12 @@
 
 import asdf
 from asdf import generic_io, versioning
+from asdf._resolver import Resolver, ResolverChain
```

Comment:
I meant for my comment to be here. Please move this to a local import in `assert_extension_correctness`. Since that is the only method that uses these in this module anyway, I would rather remove the public API all together from this.

Note `assert_estension_correctness` will need to be deprecated with the rest of the legacy extension stuff.

Reply:
Thanks for this catch. I made the change.

I'll open another PR for the `assert_extension_correctness` deprecation.